frontend/src/components/VExternalSearch/VExternalSearchForm.vue

```diff
@@ -151,6 +163,17 @@ export default defineComponent({
       emit: emit as SetupContext["emit"],
     })
 
+    const eventedOnTriggerClick = () => {
+      if (!isVisible.value) {
+        sendCustomEvent("VIEW_EXTERNAL_SOURCES", {
+          searchType: searchStore.searchType,
```

Comment:
```suggestion
          searchType: mediaStore.searchType,
```

This would change the code to match the comment above that Dhruv pointed out as incorrect... however it doesn't appear to be necessary. When I trigger the event on an all media search it already sends the search type as `all` rather than something else. I've confirmed it does this even if I switch to a specific media type then back and both when the page is rendered in SSR or client side (i.e., from the home page).